lib/tasks/local_authority_creation.rake

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require "optparse"
+
+namespace :local_authority do |args|
```

Comment:
Do we want to raise any argument errors if a required parameter is not provided?

Also, perhaps overkill, but a spec might be good here where we test running the rake task and then just see the local authority output at the end

Reply:
That's not required as we're relying on database validation errors 😄